10-algorithms.Rmd

```diff
@@ -148,7 +147,7 @@ It helps to further break-down this approach into discrete tasks before writing
 <!-- 5. Return the result -->
 
 These steps may sound straightforward, but converting words into working code requires some work and plenty of trial-and-error, even when the inputs are constrained.^[
-The algorithm will only work for *convex polygons*, which contain no internal angles greater than 180° --- no star shapes allowed.
+The algorithm will only work for *convex polygons*, which contain no internal angles greater than 180° --- no star shapes allowed. See packages **decido** or **sfdct** for approaches to non-convex polygons using external libraries. 
```

Comment:
Thanks for fixing that big omission. We should also add here a link to this (in progress) vigette that shows how you can make those triangles way faster with decido: https://geocompr.github.io/geocompkg/articles/algorithm.html

The result of rewriting the chapter: initially it used decido but a reviewer rightly pointed out that added complexity and increased the probability of confusion! This comment is the ideal place. Only question: does this deserve to be in the main text? I suspect so.